packages/mobile/src/paymentRequest/PaymentRequestListItem.tsx

```diff
@@ -99,7 +100,7 @@ export class PaymentRequestListItem extends React.Component<Props> {
           title={requester.displayName}
           ctas={this.getCTA()}
           roundedBorders={false}
-          callout={<NotificationAmount amount={this.props.amount} />}
+          callout={<NotificationAmount amount={multiplyByWei(new BigNumber(this.props.amount))} />}
```

Comment:
Nit: no need to parse this.props.amount into a new BigNumber here b.c. multiplyByWei does that already and accepts strings